src/clojure/clojurewerkz/spyglass/client.clj

```diff
@@ -56,6 +56,11 @@
       (.setProtocol cfb ConnectionFactoryBuilder$Protocol/BINARY))
     (.build cfb)))
 
+(defn- cas-value->map
+  "Given a CASValue, transform it to a map with keys #{:cas :value}"
+  [^CASValue cas-value]
+    {:value (when cas-value (.getValue cas-value))
```

Comment:
The `when`'s here are to avoid a `NullPointerException` if given a nil `cas-value`, which was borrowed from the `gets` implementation. 

I didn't confirm whether spymemcached guarantees non-null responses for fns that document a `CASValue` return type, but it seemed safer to not assume that.

I considered whether the return for a not-found key should be `nil` or `{:cas nil :value nil}`, and went with the latter for consistency with the previous `gets` implementation.